test-resources/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
it would be awesome if you provide ready to use instruction on how to use this docker config at README or wiki page